nodeJS/express/express_forms_validation.md

```diff
@@ -0,0 +1,520 @@
+### Introduction
+
+This lesson will cover how to secure your Node.js / Express applications by using the `express-validator` module to perform validation and sanitization. Proper form handling is how we maintain data integrity and prevent security vulnerabilities in our web applications.
+
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- Understanding HTML form elements and their actions.
+- The form handling process in a client-server application.
+- Using `express-validator` for validation and sanitization.
+- Best practices for form design and data integrity.
+
+### HTML forms overview
+
+First, a brief refresher on the actual HTML.
+
+Let's create a simple HTML form, with a single text field for collecting a full name, and its associated label element.
+
+The HTML structure would look something like this:
+
+```html
+<!-- The action /users/create corresponds to a route you'll define in your Express application -->
```

Comment:
We already explain what `action` attribute is below, don't think we need to do it here.
```suggestion
```